customers/views.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from django.views.generic import TemplateView, FormView
+from django.views.generic.edit import UpdateView
+from django.core.urlresolvers import reverse_lazy
+from datetime import datetime
+from card.models import Card
+from customers.forms import UserForm
+from customers.models import Customer
+
+
+class UsersIndex(TemplateView):
+    template_name = 'Users/user.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(UsersIndex, self).get_context_data(**kwargs)
+        context['users'] = Customer.objects.all()
+        return context
+
+
+class AddUser(FormView):
+    template_name = 'Users/add.html'
+    form_class = UserForm
+    success_url = reverse_lazy('cards_app:add')
+
+    def form_valid(self, form):
+        form.save()
+        return super(AddUser, self).form_valid(form)
+
+
+class UsersShow(TemplateView):
+    template_name = 'Users/show.html'
+
+    def how_many_days(self, date_finish_card):  # liczba dni nie bedzie ujemna
```

Comment:
rozumiem że jest Ci to potrzebne do wyświetlania czegoś w templatce, może spróbuj użyć to jako templatetag (podobny przykład użycia http://www.pfinn.net/custom-django-filter-tutorial.html)